src/main/resources/assets/admin/common/js/form/set/optionset/FormOptionSetOptionView.ts

```diff
@@ -103,8 +103,15 @@ export class FormOptionSetOptionView
             this.appendChild(this.optionItemsContainer);
         }
 
+        const isDefaultSelected: boolean =
```

Comment:
"`isDefaultSelected`" is misleading. It's not selected yet. We do a check to find out if it should be selected, then call `setSelected()` if it should. The most fitting is "`isDefaultOption`" (as it's declared as default in the xml schema).

Reply:
Yeah, let's go for `isDefaultAndNew`